tests/pytorch/kernel/test_rms_norm.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+
+class TestRMSNorm:
+
+    @pytest.fixture(scope='class')
+    def input(self):
+        yield torch.rand(4, 8, dtype=torch.float16, device='cuda')
+
+    @pytest.fixture(scope='class')
+    def weight(self):
+        yield torch.rand(8, dtype=torch.float16, device='cuda')
```

Comment:
I see dtypes are handled in the kernel, how about testing with some bfloat16/float32 dtype?

Reply:
Done